indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -28,8 +28,19 @@
             openInDialog: false,
             // The title for the category navigator dialog
             dialogTitle: $T.gettext("Select a category"),
-            // Restrict action to categories with no subcategories
-            selectLeafOnly: false,
+            // Allow action to specific categories
+            enableActionOn: {
+                categoriesWithSubcategories: true,
+                categoriesWithEvents: true,
+                self: false
+            },
+            disabledActionMessage: {
```

Comment:
Valid point. However, when calling the widget it will look a bit uglier than it is currently.
```js
$('<div>').categorynavigator({
    openInDialog: true,
    enableActionOn: {
        categoriesWithEvents: {
            disabled: false
        }
    },
    ...
});
```
vs
```js
$('<div>').categorynavigator({
    openInDialog: true,
    enableActionOn: {
        categoriesWithEvents: false
    },
    ...
});
```